### PR TITLE
chore: Remove login and signup related env variables from client

### PR DESCRIPTION
**Pull request by sharat87**
Remove form login and form signup env variables from client, and get this information, tenant-specific, from the server.

### CODE REVIEW

File: app/client/src/pages/UserProfile/General.tsx [hunk 1]
```diff
@@ -19,9 +19,8 @@ import {
 import { logoutUser, updateUserDetails } from "actions/userActions";
 import UserProfileImagePicker from "./UserProfileImagePicker";
 import { Wrapper, FieldWrapper, LabelWrapper } from "./StyledComponents";
-import { getAppsmithConfigs } from "@appsmith/configs";
 import { ANONYMOUS_USERNAME } from "constants/userConstants";
-const { disableLoginForm } = getAppsmithConfigs();
+import { getIsFormLoginEnabled } from "../../ce/selectors/tenantSelectors";
```
Comment (ankitakinger) [inline app/client/src/pages/UserProfile/General.tsx:23]: Please update the import here to use `@appsmith/selectors/tenantSelectors` instead

Comment (sharat87) [reply app/client/src/pages/UserProfile/General.tsx:23]: Oh damn... I'm still missing these. Sorry, on it now.

**Review by ankitakinger — APPROVED** [review]
LGTM. This will also require a PR on EE though. Mentioning it just in case we don't already have one.

**Review by ankitakinger — APPROVED** [review]

**Comment by sharat87** [comment]
Approval for test failures in this [Slack thread](https://theappsmith.slack.com/archives/C0134BAVDB4/p1684220127356489?thread_ts=1684215437.775099&cid=C0134BAVDB4).

**Pull request merged**